.circleci/config.yml

```diff
@@ -149,6 +149,7 @@ jobs:
           name: Run API Compliance
           command: >
             VER=$(cat ./version.txt)
+
```

Comment:
does this make a difference?

Reply:
Yes. folded block style (https://yaml.org/spec/1.2-old/spec.html#id2796251) in yaml treats consecutive lines as a single line. In order to treat these commands as separate commands, they must be separated by a blank line.

Reply:
ok, good to know :)